dependent/academic-psychiatry.csl

```diff
@@ -2,17 +2,15 @@
 <style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
   <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer -->
   <info>
-    <title>Modern Rheumatology</title>
-    <title-short>Mod Rheumatol</title-short>
-    <id>http://www.zotero.org/styles/modern-rheumatology</id>
-    <link href="http://www.zotero.org/styles/modern-rheumatology" rel="self"/>
+    <title>Academic Psychiatry</title>
+    <id>http://www.zotero.org/styles/academic-psychiatry</id>
+    <link href="http://www.zotero.org/styles/academic-psychiatry" rel="self"/>
     <link href="http://www.zotero.org/styles/springer-vancouver-brackets" rel="independent-parent"/>
     <link href="http://www.springer.com/cda/content/document/cda_downloaddocument/Key_Style_Points_1.0.pdf" rel="documentation"/>
     <link href="http://www.springer.com/cda/content/document/cda_downloaddocument/manuscript-guidelines-1.0.pdf" rel="documentation"/>
     <category citation-format="numeric"/>
-    <category field="medicine"/>
-    <issn>1439-7595</issn>
-    <eissn>1439-7609</eissn>
+    <issn>1042-9670</issn>
+    <eissn>1545-7230</eissn>
```

Comment:
The title seems to be different and the issn as well. Why should it then be the same file?